dove/src/cmd/init.rs

```diff
@@ -89,6 +89,9 @@ impl Cmd for Init {
         }
 
         if let Some(url) = &self.repository {
+            if !regex::Regex::new(r"^http(s)?://[a-z\d_\-.]+")?.is_match(&url.to_string()) {
```

Comment:
I belive `.` character on the last position inside `[` `]` brackets is wrong - it matches any character, making the whole expression meaningless.

Reply:
Thanks